tests/OSModulesTest.php

```diff
@@ -98,12 +98,8 @@ public function testOS($os, $variant, $modules)
             $results = $helper->generateTestData($this->getSnmpsim(), true);
         } catch (FileNotFoundException $e) {
             $this->fail($e->getMessage());
-
-            return;
```

Comment:
Why remove these?

I see an error that $helper might not always be defined below...

Reply:
Because its dead code since `fail()` throws, but I changed it